subset/security/tlstest/src/main/java/Certs.java

```diff
@@ -45,7 +45,7 @@ public void get_certificate() {
           } catch (CertificateExpiredException cee) {
             certificateReport += "\nCertificate is expired";
           } catch (CertificateNotYetValidException e) {
-        	  System.err.println("X509Certificate CertificateNotYetValidException:" + e.getMessage());
+            System.err.println("X509Certificate CertificateNotYetValidException:" + e.getMessage());
```

Comment:
This logic doesn't seem right. Why is this case different than the others?  Seems like all should be handled roughly the same.

Reply:
Same below too -- the "System.err" will be silently lost.